data/persons.txt

```diff
@@ -107,14 +107,29 @@ person "Cap'n Pester"
 
 
 
-person "Marauding Max K"
-	government "Author"
-	frequency 300
+person "Marauding Max"
+	government "Indigenous Lifeform"
```

Comment:
Well now the Archons get pissed if you capture Max.

Reply:
according to government.txt, there are no consequences for killing members of the "Indigenous Lifeform" government.

`government "Indigenous Lifeform"
	# Nothing you do permanently angers indigenous creatures, because they are
	# not sentient and do not remember you as an individual.
	"player reputation" 1
	"penalty for"
		assist 0
		disable 0
		board 0
		capture 0
		destroy 0
		atrocity 0`

Reply:
still not as bad as PotP, lol